ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -114,15 +114,17 @@ public CompletableFuture<FastSyncState> findPivotBlock(
   protected CompletableFuture<FastSyncState> handleFailure(final Throwable error) {
     trailingPeerRequirements = Optional.empty();
     Throwable rootCause = ExceptionUtils.rootCause(error);
-    if (rootCause instanceof SyncException) {
+    if (rootCause instanceof NoSyncRequiredException) {
+      return CompletableFuture.completedFuture(new NoSyncRequiredState());
+    } else if (rootCause instanceof SyncException) {
       return CompletableFuture.failedFuture(error);
     } else if (rootCause instanceof StalledDownloadException) {
-      LOG.debug("Stalled sync re-pivoting to newer block.");
+      LOG.info("Stalled sync re-pivoting to newer block.");
       return start(FastSyncState.EMPTY_SYNC_STATE);
     } else if (rootCause instanceof CancellationException) {
       return CompletableFuture.failedFuture(error);
     } else if (rootCause instanceof MaxRetriesReachedException) {
-      LOG.debug(
+      LOG.info(
```

Comment:
are we sure this log is not spamming during the sync. maybe we need to start a sync on mainnet to be sure we are not used debug to avoid log spamming

Reply:
@non-fungible-nelson did you see any issues with your holesky sync?

Reply:
I've reverted it to a `debug` log